tests/config.rs

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pub mod util;
+
+use assert_cmd::prelude::*; // Add methods on commands
+
+#[tokio::test]
+async fn test_config_dump() -> Result<(), Box<dyn std::error::Error>> {
+    let mut tm = util::setup_with_lock().await?;
+
+    let mut c = tm.command()?;
+    let cmd = c.args(&["config", "clear"]);
+    cmd.assert().success();
+
+    let table_name = tm.create_temporary_table("pk", None).await?;
+    assert_config_use_dump(&tm, table_name).await?;
+
+    Ok(())
+}
+
+#[tokio::test]
+async fn test_config_clear() -> Result<(), Box<dyn std::error::Error>> {
+    let mut tm = util::setup_with_lock().await?;
+
+    let mut c = tm.command()?;
+    let cmd = c.args(&["config", "clear"]);
+    cmd.assert().success();
+
+    let table_name = tm.create_temporary_table("pk", None).await?;
+    assert_config_use_dump(&tm, table_name).await?;
```

Comment:
I'm not sure these part should be included in the test for clear command because the `test_config_dump` contains the same test. If there is a reason, I think this should be commented in the test code.